### PR TITLE
Test: Filter UTXOs by lock type in getSpendables

### PR DESCRIPTION
**Pull request by omerfirmak**
```
Should fix crashes in Flash where TxBuilder tries to sign a
non-LockType.Key output
```

cc @linked0

### CODE REVIEW

File: source/agora/test/Base.d [hunk 1]
```diff
@@ -1138,7 +1139,8 @@ public class TestAPIManager
 
         if (!no_txs)
         {
-            auto utxo_pairs = first_client.getSpendables(1.coins, OutputType.Payment);
+            auto utxo_pairs = first_client.getSpendables(1.coins, OutputType.Payment,
+                agora.script.Lock.LockType.Key);
```
Comment (linked0) [inline source/agora/test/Base.d:1143]: `agora.script.Lock.LockType.Key` is default to the `lock_type` parameter on the `getSpendables`, so can't we remove the last argument?

**Review by linked0 — COMMENTED** [review]
Thanks, this is a time saver. There's one request for the code.

File: source/agora/test/Base.d (hunk 1 above)
Comment (omerfirmak) [reply source/agora/test/Base.d:1143]: Sadly no, `RemoteAPI` (LocalRest) generator does not work with default values.

Comment (linked0) [reply source/agora/test/Base.d:1143]: I see.

**Review by linked0 — APPROVED** [review]

**Pull request merged**